src/Ledger/Ratify.lagda

```diff
@@ -142,17 +253,56 @@ module _ (Γ : RatifyEnv) (cc : CCData) (votes : (GovRole × Credential) ⇀ Vot
                                            _            → Vote.no) ❵ᵐ
 
   actualDRepVotes : VDeleg ⇀ Vote
-  actualDRepVotes = mapKeys (uncurry credVoter) (roleVotes GovRole.DRep) (λ where _ _ refl → refl)
-                  ∪ᵐˡ constMap (map (credVoter DRep) activeDReps) Vote.no
+  actualDRepVotes =  mapKeys (uncurry credVoter) (roleVotes GovRole.DRep)
+                     (λ where _ _ refl → refl)
+                     ∪ᵐˡ constMap (map (credVoter DRep) activeDReps) Vote.no
     where
       activeDReps : ℙ Credential
       activeDReps = dom (filterᵐ (to-sp (currentEpoch ≤ᵉ?_ ∘ proj₂)) dreps ˢ)
 
   actualVotes : VDeleg ⇀ Vote
   actualVotes = mapKeys (credVoter CC) actualCCVotes (λ where _ _ refl → refl)
               ∪ᵐˡ actualPDRepVotes ∪ᵐˡ actualDRepVotes
-              ∪ᵐˡ mapKeys (uncurry credVoter) votes (λ where _ _ refl → refl) -- TODO: make `actualVotes` for SPO
-
+              ∪ᵐˡ mapKeys (uncurry credVoter) votes (λ where _ _ refl → refl)
+              -- TODO: make `actualVotes` for SPO
+\end{code}
+} %% end small
+\caption{%Ratify i:
+Types and proofs for the ratification of governance actions}
+\label{fig:defs:ratify-i}
+\end{figure*}
+The code in Figure~\ref{fig:defs:ratify-i} defines some of the types required for ratification of a governance action.
+\begin{itemize}
+  \item Module parameters: \AgdaBound{Γ} is a ratification environment, \AgdaBound{cc} contains constitutional committee data,
```

Comment:
I like the latter as I think module parameters are best avoided unless their presence should be more or less global.  However, I'll stick with your first suggestion for now since this PR is about the prose/doc, and wait for a future code improvement opportunity to possibly refine the code.